shim.php

```diff
@@ -10,6 +10,44 @@ function toXPath($cssExpr, $prefix = 'descendant-or-self::') {
     }
 }
 
+// Add aliases for PHPUnit 6
+
+namespace {
+    if (version_compare(PHP_VERSION, '7.0.0') >= 0 && class_exists('PHPUnit\Framework\TestCase')) {
```

Comment:
It was a `forward compatibility layer`, wasn't it? that's why you need to check `PHPUnit_Framework_TestCase`.